packages/extension/src/components/molecules/XRPLTransaction/XRPLTransaction.tsx

```diff
@@ -115,14 +144,51 @@ export const XRPLTransaction: FC<XRPLTxProps> = ({
     );
   };
 
+  const renderCurrency = (params: { title: string; value: Currency }) => {
+    const { title, value } = params;
+    const currency =
+      value.currency.length === 40 ? convertHexCurrencyString(value.currency) : value.currency;
+    const formatted = value.issuer ? `${currency}\n${value.issuer}` : currency;
```

Comment:
currency is used only one time, so no need to put in a variable:
```suggestion
    const formatted = value.issuer ? `${currency}\n${value.issuer}` : convertHexCurrencyString(value.currency);
```

Reply:
currency appears twice?

Reply:
Oh right!